javascript-typing-tutor/main.js

```diff
@@ -0,0 +1,22 @@
+var $letters = document.querySelectorAll('span');
+var $spanLetter = [];
+for (var i = 0; i < $letters.length; i++) {
+  $spanLetter.push($letters[i].textContent);
+}
+var currentLetter = 0;
+
+document.addEventListener('keydown', function (event) {
+  if (currentLetter >= $spanLetter.length) {
+    return;
+  }
+  if (event.key === $spanLetter[currentLetter] || (event.code === 'Space' && (currentLetter === 6 || currentLetter === 14 || currentLetter === 19 || currentLetter === 25))) {
```

Comment:
This seems like a really manual way to deal with spaces, what if you have spaces in other positions of the text? I would try to find another way to deal with spaces in the text.